pages/index.vue

```diff
@@ -1,20 +1,139 @@
 <template>
   <div>
-    <KiboWidgetFactory :document-name="pageName" />
+    <SfHero class="hero" :slider-options="{ autoplay: false }">
+      <SfHeroItem
+        v-for="(img, index) in heroes"
+        :key="index"
+        :image="img.image"
+        :title="img.title"
+        :subtitle="img.subtitle"
+        :button-text="img.buttonText"
+        :background="img.background"
+        :class="img.className"
+      >
+        <template #subtitle="{ subtitle }">
+          <div :class="{ 'display-none': !subtitle }" class="sf-hero-item__subtitle">
+            {{ subtitle }}
+          </div>
+        </template>
+        <template #title="{ title }">
+          <div :class="{ 'display-none': !title }" class="sf-hero-item__title">{{ title }}</div>
+          <div :class="{ 'display-none': !img.description }" class="sf-hero-item__description">
+            {{ img.description }}
+          </div>
+        </template>
+        <template #call-to-action="{ buttonText, link }">
+          <div v-if="buttonText" class="sf-hero-item__button">
+            <SfButton :link="link" data-testid="hero-cta-button">
+              {{ buttonText }}
+            </SfButton>
+          </div>
+        </template>
+      </SfHeroItem>
+    </SfHero>
   </div>
 </template>
 
 <script>
-import KiboWidgetFactory from "@/components/cms/KiboWidgetFactory.vue"
+import { SfHero, SfButton } from "@storefront-ui/vue"
+import CarouselImage1 from "@/assets/images/Mobile-Homepage-HeroBanner.png"
 
 export default {
-  components: { KiboWidgetFactory },
+  components: { SfHero, SfButton },
   layout: "full-width",
   setup() {
     const pageName = "home-page"
+    const heroes = [
+      {
+        title: "Save upto 50%",
+        subtitle: "Check Off Your List Event",
+        description: "Shop early to get your holiday gifts on time.",
+        buttonText: "Shop Holiday Items on Sale",
+        background: "rgb(236, 239, 241)",
+        image: CarouselImage1,
+      },
+      {
+        title: "Save upto 70%",
+        subtitle: "Check Off Your List Event",
+        description: "Shop early to get your holiday gifts on time.",
+        buttonText: "Shop Holiday Items on Sale",
+        background: "rgb(236, 239, 241)",
+        image: "https://i.pinimg.com/474x/ac/0f/38/ac0f388b725f5c24d4f0b63c547be9f5.jpg",
+        // className: "sf-hero-item--position-bg-top-left sf-hero-item--align-right",
```

Comment:
Is this commented code need to keep here?

Reply:
Removed